buildkit/pg_cron.yaml

```diff
@@ -8,7 +8,9 @@ description: Run periodic jobs in PostgreSQL.
 licence: PostgreSQL
 keywords:
   - cron jobs
-  - periodic jobs
+  - periodic
+  - scheduling
+  - time
```

Comment:
If you want, just copy how the authors tag them on GithHub, for example for this one:

![Screenshot 2023-11-15 at 4 17 58 PM](https://github.com/pgxman/buildkit/assets/169064/f6d38172-54dc-45a3-b583-218a9d14c694)

Same for other extensions